packages/manager/apps/key-management-service/src/components/credential/credentialStatus/credentialStatus.component.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  ODS_CHIP_SIZE,
+  ODS_TEXT_COLOR_INTENT,
+  OdsChipAttribute,
+} from '@ovhcloud/ods-components';
+import { OsdsChip } from '@ovhcloud/ods-components/react';
+import { OkmsCredentialStatus } from '@/types/okmsCredential.type';
+
+export type KeyStatusProps = Omit<OdsChipAttribute, 'color'> & {
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  state: OkmsCredentialStatus | (string & {});
+};
+
+export const CredentialStatus = ({ state, ...props }: KeyStatusProps) => {
+  const { t } = useTranslation('key-management-service/credential');
+
+  const { size, ...otherProps } = props;
```

Comment:
You can add the default value for size directly in the props declaration
```suggestion
export const CredentialStatus = ({ state, size = ODS_CHIP_SIZE.md, ...otherProps }: KeyStatusProps) => {
  const { t } = useTranslation('key-management-service/credential');
```